includes/template-functions.php

```diff
@@ -876,6 +892,17 @@ function sensei_the_single_lesson_meta() {
 		?>
 		<section class="lesson-meta">
 			<?php
+			/**
+			 * Filters the position of the video in the lesson meta.
+			 *
+			 * Fires before the lesson meta is displayed.
```

Comment:
It seems to also be documented on `templates/single-lesson.php`.

Again, I didn't check our generated documentation, so I'm not sure if the text "Fires before the lesson meta is displayed" also applies to the other case. If not, could it cause confusion depending on how our documentation is generated?